src/test/java/org/ays/tests/institution/adminregistrationmanagementservice/PostAdminRegistrationApplicationRejectTest.java

```diff
@@ -46,7 +46,7 @@ public void rejectANotCompletedApplication() {
                         containsString("WAITING")));
     }
 
-    @Test(dataProvider = "invalidRejectReason")
+    @Test(dataProvider = "invalidRejectReason",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html